includes/event/event-cpt.php

```diff
@@ -46,13 +46,14 @@ function create_event_cpt()
         'menu_position' => 5,
         'show_in_admin_bar' => true,
         'show_in_nav_menus' => true,
+        'show_in_rest' => true,
```

Comment:
I'm curious to know how this will be used? Any plan for a SPA in mind? :)

Reply:
If they do not appear in the rest api, the query loop gutenberg block cannot access any CPT.

Reply:
Ah I see, then great catch!